packages/plugin-flex/src/commands/flex/plugins/release.ts

```diff
@@ -56,20 +56,63 @@ export default class FlexPluginsRelease extends CreateConfiguration {
     }),
   };
 
+  public _flags: OutputFlags<typeof FlexPluginsRelease.flags>;
+
   // @ts-ignore
   private prints;
 
-  private _parsed?: ParserOutput<any, any>;
-
   constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
     super(argv, config, secureStorage, { runInDirectory: false });
 
     this.scriptArgs = [];
     this.prints = this._prints.release;
+    this._flags = {
+      json: false,
+      'clear-terminal': false,
+      region: '',
+      'configuration-sid': '',
+      name: '',
+      plugin: '',
+      'disable-plugin': '',
+      'enable-plugin': '',
+      description: '',
+      new: false,
+    };
   }
 
   async init(): Promise<void> {
-    this._parsed = await this.parseCommand(FlexPluginsRelease);
+    const parsed: ParserOutput<any, any> = await this.parseCommand(FlexPluginsRelease);
+    if (parsed.flags[configurationSidFlex]) {
+      this._flags = parsed.flags;
+      return;
+    }
+
+    [descriptionFlex, nameFlex].forEach((key) => {
```

Comment:
Why are you doing this? Oclif supports these `exclusive` etc already. If it dones't, the validation should be done inside the `parseCommand` not in here.

Reply:
This logic was already present in the getter `_flags()`. I have just moved it to `init` as suggested. However, we can get rid of it I believe. Since I am not aware of why it was added in the first place, we have decided to take it up as part of [FLEXY-4154](https://issues.corp.twilio.com/browse/FLEXY-4154) when we have more clarity.